book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = rentals
+  end
+end
```

Comment:
- [ ] You should create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. They should be implemented similar to the following:

* A book has many rentals:

```ruby
class Book

  attr_accessor :title, :author, :rentals
  .
  .

  def initialize(title, author)
    .
    .
    # *@rentals should be initialized.
    @rentals = []
  end
  
  # *A method for adding rentals should be present.
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
end
```